src/Network/WebSockets/Http.hs

```diff
@@ -238,6 +239,11 @@ getRequestSubprotocols rh = maybe [] parse mproto
         mproto = lookup "Sec-WebSocket-Protocol" $ requestHeaders rh
         parse = filter (not . B.null) . BC.splitWith (\o -> o == ',' || o == ' ')
 
+--------------------------------------------------------------------------------
+-- | Get the @Sec-WebSocket-Extensions@ header
+getRequestSecWebSocketExtensions :: RequestHead -> Maybe B.ByteString
+getRequestSecWebSocketExtensions p =
+    lookup "Sec-WebSocket-Extensions" (requestHeaders p)
```

Comment:
Semantically, I think this field is supposed to be more like a list than like a `Maybe`.  Would you mind changing it to that?